### PR TITLE
Rostering api

### PR DESCRIPTION
**Pull request by SibiAkkash**

### CODE REVIEW

File: server/schema.py [hunk 1]
```diff
@@ -632,6 +633,31 @@ class AddOrRemoveTeamRegistrationSchema(Schema):
...
+    registration_id: int
+
+
+class TournamentPlayerRegistrationSchema(Schema):
+    team: TeamSchema
+    player: PlayerSchema
+
+    # @staticmethod
+    # def resolve_role_full(registration: Registration) -> str:
+    #     return Registration.Role._value2member_map_[registration.role].label
```
[the remark this answers is not in the document]
Comment (SibiAkkash) [reply server/schema.py:654]: Yes. Currently we are not using that schema to populate anything in UI. I've added that line in the fetch roster endpoint so we get full role name.

[the remark this answers is not in the document]
Comment (SibiAkkash) [reply server/schema.py:654]: should i remove for now ?

[the remark this answers is not in the document]
Comment (Joe2k) [reply server/schema.py:654]: oh ok keep keep

**Comment by SibiAkkash** [comment]
there was an unused tournament_slug query param in `add to roster` endpoint, i've removed that and pushed

**Pull request merged**